cedar-policy/src/api.rs

```diff
@@ -2228,7 +2139,7 @@ impl PolicySet {
 
     /// Build the [`PolicySet`] from just the AST information
     #[cfg_attr(not(feature = "protobufs"), allow(dead_code))]
-    fn from_ast(ast: ast::PolicySet) -> Result<Self, PolicySetError> {
+    pub(crate) fn from_ast(ast: ast::PolicySet) -> Result<Self, PolicySetError> {
```

Comment:
Just checking: Do we want to keep the `cfg_attr(not(feature = "protobufs")` here?

Reply:
This says "if protobufs is not enabled, allow(dead_code)".  We do still need that I believe.